### PR TITLE
remove ip information

### PR DESCRIPTION
**Pull request by QingyangKong**
remove test net IP information from codes.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -42,3 +42,6 @@ docs/build
 
 # OS X
 .DS_Store
+
+# example property files
+.properties
```
Comment (zhangsoledad) [inline .gitignore]: feed new line

Comment (zhangsoledad) [reply .gitignore]: https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

File: examples/src/main/resources/config.properties.example [hunk 2]
```diff
@@ -0,0 +1,2 @@
+TestNetIpAddr=http://127.0.0.1:1337
+ChainId=1
```
Comment (zhangsoledad) [inline examples/src/main/resources/config.properties.example]: :green_apple:

**Comment by QingyangKong** [comment]
Sorry for that, I've already add setting in my IDE to force to feed newline in the end on save.

**Comment by zhangsoledad** [comment]
:heart: :green_heart: :purple_heart: :yellow_heart: :black_heart:

**Pull request merged**